### PR TITLE
added: Support for specifying source hosts in PARASITIC_NET_HOST_xxxx…

### PR DESCRIPTION
**Pull request by arnova**
…xxxxxxxxx variables

### CODE REVIEW

**Comment by arnova** [comment]
@abelbeck : Mind reviewing this?

File: share/arno-iptables-firewall/environment [hunk 1]
```diff
@@ -1024,6 +1024,32 @@ parse_rule()
...
+    shosts=$(get_hosts_ih "$left_rule" "$ANYHOST")
+    dhosts=$(get_hosts_hp "$right_rule")
+
+    if [ -z "$dhosts" ]; then
+      parse_rule_warning "$rule"
+      return 1
+    fi
+    ;;
+
+  shosts-dhosts:ANYHOST)
```
Comment (abelbeck) [inline share/arno-iptables-firewall/environment]: @arnova : This `shosts-dhosts:ANYHOST` does not appear to be used.

Comment (arnova) [reply share/arno-iptables-firewall/environment]: Correct: It's for consistency and possible future use. It prevents breaking API when it's required by some plugin at some point in time.

Comment (abelbeck) [reply share/arno-iptables-firewall/environment]: OK, fair enough.

**Pull request merged**